engine/src/main/java/com/google/android/fhir/FhirEngine.kt

```diff
@@ -71,6 +71,13 @@ interface FhirEngine {
    * result of the [download] operation.
    */
   suspend fun syncDownload(download: suspend (SyncDownloadContext) -> List<Resource>)
+
+  /**
+   * Total count of entities available for given search
```

Comment:
available is misleading. this is the count of resources returned for a given search.

```suggestion
   * Total count of resources for given search
```